lib/src/Uncertainty/Process/GaussianProcess.cxx

```diff
@@ -145,8 +145,8 @@ void GaussianProcess::initialize() const
     // Scaling factor of the matrix : M-> M + \lambda I with \lambda very small
     // The regularization is needed for fast decreasing covariance models
     Scalar maxEV = -1.0;
-    const Scalar startingScaling = ResourceMap::GetAsScalar("GeneralLinearModelAlgorithm-StartingScaling");
-    const Scalar maximalScaling = ResourceMap::GetAsScalar("GeneralLinearModelAlgorithm-MaximalScaling");
+    const Scalar startingScaling = ResourceMap::GetAsScalar("GaussianProcess-StartingScaling");
+    const Scalar maximalScaling = ResourceMap::GetAsScalar("GaussianProcess-MaximalScaling");
```

Comment:
Good catch! Looks like the "StartingScaling" ResourceMap keys are all equal to 1e-13 by default, though, so it is unlikely many users were affected by this bug.

Reply:
yes but if some users update the key, it will be useless ;-)